code/datums/spells/ethereal_jaunt.dm

```diff
@@ -15,44 +15,75 @@
 
 	var/jaunt_duration = 50 //in deciseconds
 	var/jaunt_in_time = 5
+
+	///Do we do the glowy wizard thing on the tile the wizard entered / exit
+	var/has_jaunt_effect = TRUE
 	var/jaunt_in_type = /obj/effect/temp_visual/wizard
 	var/jaunt_out_type = /obj/effect/temp_visual/wizard/out
+	/// Do we show the fun blue water effect spread out thing
+	var/has_smoke_jaunt_effect = TRUE
+
+	/// Sounds to play
```

Comment:
Documentation is per-var not for block level things. This will tag only `jaunt_enter_sound` as "Sounds to play", which is misleading given its one sound, not a list.

Reply:
Ah, will change it, thank you.